Source/Scene/Primitive.js

```diff
@@ -812,6 +820,11 @@ define([
                     this._state = PrimitiveState.COMBINING;
 
                     when(promise, function(packedResult) {
+                        if (!defined(packedResult)) {
+                            that._state = PrimitiveState.FAILED;
```

Comment:
Everything else looks fine; I just want to better understand what the change in behavior is.  If I create a PolylineGeometry with 3 identical points, the Primitive will no longer throw, but it looks like the state becomes `failed`?  Does that also mean `Primitive.ready` will never be true either?